packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -121,8 +135,9 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                             <ContractAuditItem
                                 id='dt_bt_label'
                                 icon={<Icon icon='IcContractPayout' size={24} />}
-                                label={localize('Payout per point')}
+                                label={vanilla_payout_text}
                                 value={`${display_number_of_contracts} ${getCurrencyDisplayCode(currency)}` || ' - '}
+                                should_format={!is_vanilla}
```

Comment:
There is no such prop in master in `<ContractAuditItem />` + `show_payout_per_point` is currently used for `Vanillas` and `Turbos`. What do you want to format in Turbos)